dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -93,7 +93,14 @@ public static void main(String[] arguments)
     CommandLine commandLine = StaticLinkageCheckOption.readCommandLine(arguments);
     ImmutableList<Path> inputClasspath =
         StaticLinkageCheckOption.generateInputClasspath(commandLine);
-    logger.info("Input class path size: " + inputClasspath.size());
+
+    StringBuilder classPathOutputBuilder =
```

Comment:
Oh, sorry. I didn't mean to make you do anything nontrivial. I thought you could just do

`logger.info("Input class path: " + inputClassPath);`

Reply:
The input class path may have more than hundreds of elements. So I think multi-line look better.